driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedCrudHelper.java

```diff
@@ -1760,6 +1778,226 @@ public OperationResult createChangeStreamCursor(final BsonDocument operation) {
         });
     }
 
+    public OperationResult clientBulkWrite(final BsonDocument operation) {
+        Set<String> unexpectedOperationKeys = singleton("saveResultAsEntity");
```

Comment:
This [operation field](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.md#operation) is handled (or not) by each operation, and not by a piece of code common to all operations. It seems reasonable for all operations that don't implement it to fail if the field is encountered. I don't know why other operations don't do that. Maybe because it is not expected to be encountered for them.